### PR TITLE
add section on getting a cookie value

### PR DESCRIPTION
**Pull request by marcuspoehls**
This PR adds a section on getting a previously set cookie value.

In addition to @mtharrison’s PR https://github.com/hapijs/hapijs.com/pull/546

### CODE REVIEW

File: lib/tutorials/en_US/cookies.md [hunk 1]
```diff
@@ -53,6 +53,19 @@ The `state()` method is also available on the [response object](/api#response-ob
...
+
+The `request.state` object contains the parsed HTTP state. Each key represents the cookie name and its value is the defined content.
+
+```javascript
+const value = request.state.data
+// console.log(value) will give you { firstVisit : false }
+```
+
+The sample code uses the `data` cookie key from above where the related value is set to `{ firstVisit: false }`.
+
```
Comment (devinivy) [inline lib/tutorials/en_US/cookies.md]: I believe the "Overriding options" subsection below is intended to go below the "Setting a cookie" section.

File: lib/tutorials/en_US/cookies.md [hunk 2]
```diff
@@ -53,6 +53,19 @@ The `state()` method is also available on the [response object](/api#response-ob
...
 ```
 
+## Getting a cookie value
+
+Access a cookie’s value via `request.state` in a route handler, pre-requisite, or request lifecycle extension point.
+
+The `request.state` object contains the parsed HTTP state. Each key represents the cookie name and its value is the defined content.
+
+```javascript
+const value = request.state.data
```
Comment (mtharrison) [inline lib/tutorials/en_US/cookies.md]: Lint: Missing semicolon here

Comment (marcuspoehls) [reply lib/tutorials/en_US/cookies.md]: Thanks

File: lib/tutorials/en_US/cookies.md (hunk 1 above)
Comment (marcuspoehls) [reply lib/tutorials/en_US/cookies.md]: Yep, you’re right.

**Comment by mtharrison** [comment]
Thanks!

**Comment by marcuspoehls** [comment]
:handshake: thanks!

**Pull request merged**